app/Http/Controllers/Api/StatsController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Contracts\Controller;
+use App\Http\Resources\Stats;
+use App\Models\Enums\PirepState;
+use App\Models\Pirep;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class StatsController extends Controller
+{
+    public function index(Request $request)
+    {
+        $where = [];
+        $where['user_id'] = Auth::id();
+        $where['state'] = PirepState::ACCEPTED;
+        $avgStats = ['flight_time', 'landing_rate', 'fuel_used', 'score'];
+        $response = [];
+
+        $response['flights'] = Pirep::where($where)->count();
+        $response['flight_time'] = Pirep::where($where)->count();
+
+        foreach ($avgStats as $static) {
+            $response['average_'.$static] = Pirep::where($where)->avg($static);
+        }
+
+        $response['balance'] = Auth::user()->journal->balance->getValue() ?? 0;
```

Comment:
If you don't want to create extra queries, you should eager load the journal relationship here.

Also no need to call the Auth facade twice, you can get the user once with relationships and use wherever needed 😉